core/domain/exp_domain_test.py

```diff
@@ -629,16 +635,37 @@ def test_validation(self):
         init_state.update_interaction_default_outcome(default_outcome.to_dict())
         exploration.validate()
 
+        interaction.solution = state_domain.Solution.from_dict(
```

Comment:
Can we use the interface update_intraction_solution to change this value?

Reply:
I think the update_intraction_solution would raise an error as we are passing an invalid exploration? We just need to validate the exploration and this explicitly shows that we are assigning an invalid solution

Reply:
done!